components/zcash_protocol/src/consensus.rs

```diff
@@ -590,7 +582,6 @@ impl From<BranchId> for u32 {
             BranchId::Heartwood => 0xf5b9_230b,
             BranchId::Canopy => 0xe9ff_75a6,
             BranchId::Nu5 => 0xc2d6_d0b4,
-            #[cfg(zcash_unstable = "nu6")]
             BranchId::Nu6 => 0xc8e7_1055,
```

Comment:
We can't put `None` here, should we go with `0xffff_ffff` or a random one that is not `0xc8e7_1055`?

`0xc8e7_1055` is the one in the proposed ZIP: https://github.com/zcash/zips/pull/878/files#diff-7912352c4bcac5fd61ea274ab3d1f0d53bf48f2bbb7426ede47b5982d8dc564bR44

Reply:
As long as we haven't set any activation heights for mainnet or testnet, we can use the random one from the current ZIP draft while getting the consensus rules implemented. Once we set an activation height, any subsequent consensus rule change would force us to change this.

Reply:
Ok, maybe is safer to introduce a new branch id here?

```suggestion
            // TODO: Change this to ZIP-253 value when setting activation heights.
            BranchId::Nu6 => 0xffff_fffe,
```

Reply:
We discussed this between the ECC and the ZF engineers and decided to use the same branch ID as the current open ZIP(https://github.com/zcash/zips/pull/878/files), which is `0xc8e7_1055`.